tests/unit/archiving/test_import.py

```diff
@@ -173,9 +173,8 @@ def test_import_with_targz_extension(self):
         pr_imp.unpack(
             origin_path=pack_path_comp, csv_file_name=pack_path_csv, compress=True
         )
-        # here the 7 is the length of '.tar.gz' string
-        with tarfile.open(pack_path_comp[:-7] + ".tar.gz", "r:gz") as tar:
-            tar.extractall()
+        with tarfile.open(pack_path_comp, "r:gz") as tar:
+            tar.extractall(path=pack_path_comp[:-7])
```

Comment:
```suggestion
            tar.extractall(path=pack_path_comp[:-len(".tar.gz")])
```

I just checked in a notebook and was happy to see this trick works. Cute way to get rid of a magic number, IMO